index.html

```diff
@@ -2,17 +2,65 @@
 layout: default
 permalink: "/"
 ---
-<div class="mission-statement">
-    <h2>Our Mission</h2>
-    <p>Accidental Shakespeare is an ensemble theater company that thoughtfully explores challenging plays, to forge connections between classical themes and contemporary culture in intimate settings.</p>
-    <!-- need to decide where Get Involved links to -->
-    <form action="/join" class="button">
-        <input type="submit" value="Get Involved" />
-    </form>
+
+<div id="main_area">
+<div class="row">
+  <div class="col-xs-12" id="slider">
+      <!-- Top part of the slider -->
+      <div class="row carousel-block">
+          <div class="col-sm-8" id="carousel-bounding-box">
+            <div class="carousel slide" id="myCarousel">
+            <!-- Carousel items -->
+              <div class="carousel-inner">
+                  <div class="active item" data-slide-number="0">
+                  <img src="/assets/images/IMG_0656.jpg"></div>
```

Comment:
Could you nest these divs like your're doing on L33 with the `<a>` tags for consistency?
 I.E:
```
<div class="active item" data-slide-number="0">
    <img src="/assets/images/IMG_0656.jpg">
</div>
```